src/symbolize/gimli/lru.rs

```diff
@@ -0,0 +1,73 @@
+use core::mem::{self, MaybeUninit};
+use core::ptr;
+
+/// least-recently-used cache with static size
+pub(crate) struct Lru<T, const N: usize> {
+    // SAFETY: len <= initialized values
+    len: usize,
+    arr: [MaybeUninit<T>; N],
+}
+
+impl<T, const N: usize> Default for Lru<T, N> {
+    fn default() -> Self {
+        Lru {
+            len: 0,
+            arr: [const { MaybeUninit::uninit() }; N],
```

Comment:
this requires an MSRV bump, *or* to use the infamously sus-looking-but-actually-okay-for-this-ONE-instance:
```suggestion
            arr: unsafe { MaybeUninit::uninit().assume_init() },
```

Reply:
On the one hand I'm not against bumping the MSRV for this crate but on the other hand it is only one line. But on the third hand it is a very sussy line that would at least need a comment saying "no really, this is ok!"

What would the new MSRV be?

Reply:
if https://github.com/rust-lang/backtrace-rs/pull/689/commits/318dd91a3dc077bd7c3710ea66c5bc664f61590a passes then the answer is 1.79

Reply:
Ah 1.73 to 1.79? That seems fine to me. We should probably also update the `rust-version` in `Cargo.toml` as that seems to have gone out of the sink.